config/test.exs

```diff
@@ -40,3 +40,14 @@ config :remote_retro, :mock_user, %{
   "profile" => "https://plus.google.com/108658712426577966861",
   "sub" => "108658712426577966861"
 }
+
+config :remote_retro, :other_user, %{
+  "email" => "misstestuser@gmail.com",
+  "email_verified" => "true", "family_name" => "Alexander",
+  "gender" => "female", "given_name" => "Nicole",
```

Comment:
Nicole, eh? 👍

Reply:
thought we'd want to have a good selection of test users.